api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/CardClient.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.dhsc.htbhf.claimant.service;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.dhsc.htbhf.claimant.exception.CardClientException;
+import uk.gov.dhsc.htbhf.claimant.model.card.CardRequest;
+import uk.gov.dhsc.htbhf.claimant.model.card.CardResponse;
+
+/**
+ * Service for interacting with the card services api..
+ */
+@Service
+@Slf4j
+public class CardClient {
+
+    private static final String CARDS_ENDPOINT = "/v1/cards";
+    private final String cardsUri;
+    private final RestTemplate restTemplate;
+
+    /**
+     * Uses a non request scope rest template due to being outside the context of a request.
```

Comment:
Once you've changed how the RestTemplate works you'll want to say that this reflects uses the sessionId requestId from the job.